pkg/machines/components/stateIcon.jsx

```diff
@@ -62,7 +62,7 @@ export const StateIcon = ({ state, valueId, error, dismissError }) => {
             </Label>}
             <Label color={stateMap[state] && stateMap[state].color}
                    icon={stateMap[state] && stateMap[state].icon}
-                   className="vm-state-text"
+                   className={"vm-state-text vm-state--" + state.toLowerCase().replaceAll(' ', '-')}
```

Comment:
Is there a better way to do this, or is this fine?

Reply:
(The goal is to have a class name to attach some CSS to, so we can de-emphasize a shut off state.)